README.md

```diff
@@ -41,28 +41,40 @@ At this time pya is more suitable for offline rendering than realtime.
 ## Authors and Contributors
 
 * Thomas Hermann, Ambient Intelligence Group, Faculty of Technology, Bielefeld University (author and maintainer)
-* Jiajun Yang, Ambient Intelligence Group, Faculty of Technology, Bielefeld University (co-author)
+* Jiajun Yang, Holoplot GmbH (co-author)
 * Alexander Neumann, Neurocognitions and Action - Biomechanics, Bielefeld University
 * Contributors will be acknowledged here, contributions are welcome.
 
 ## Installation
 
```

Comment:
This is a bit tough to read. There is a disclaimer refers to a method I (as a reader) haven't read yet and it's also not clear whether this means the whole installation section or just item 1.  People usually do not bother much reading instructions very diligently, so this section must be easy to comprehend.